docs/en/02_Developer_Guides/01_Templates/04_Rendering_Templates.md

```diff
@@ -34,6 +34,19 @@ echo $arrayData->renderWith('Coach_Message');
 
 // returns "<strong>John</strong> is the Head Coach on our team."
 
+```
```

Comment:
This three backquotes are unnecessary

Reply:
..? They are necessary or the above code block will be unclosed.

Reply:
oh, true, overlooked that